2019-10-27 16:59:18,782	INFO	main1	<module>	Line:11	app started
2019-10-27 16:59:18,794	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-10-27 16:59:18,836	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-10-27 16:59:18,889	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-10-27 16:59:19,276	INFO	_internal	_log	Line:122	192.168.2.210 - - [27/Oct/2019 16:59:19] "GET /ProblemSet/init/a9500dc7-2b9d-42f6-900c-275b740b4dc1/qq HTTP/1.1" 200 -
2019-10-27 16:59:30,570	INFO	_internal	_log	Line:122	192.168.2.210 - - [27/Oct/2019 16:59:30] "GET /ProblemSet/a9500dc7-2b9d-42f6-900c-275b740b4dc1/ww HTTP/1.1" 200 -
2019-10-27 16:59:31,630	INFO	_internal	_log	Line:122	192.168.2.210 - - [27/Oct/2019 16:59:31] "GET /ProblemSet/init/a9500dc7-2b9d-42f6-900c-275b740b4dc1/ww HTTP/1.1" 200 -
2019-10-27 16:59:45,957	INFO	_internal	_log	Line:122	192.168.2.210 - - [27/Oct/2019 16:59:45] "GET /ProblemSet/a9500dc7-2b9d-42f6-900c-275b740b4dc1/ww HTTP/1.1" 200 -
2019-10-27 16:59:47,010	INFO	_internal	_log	Line:122	192.168.2.210 - - [27/Oct/2019 16:59:47] "GET /ProblemSet/init/a9500dc7-2b9d-42f6-900c-275b740b4dc1/ww HTTP/1.1" 200 -
2019-10-27 17:00:12,721	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
